Tools/scripts/pathfix.py

```diff
@@ -164,12 +180,28 @@ def fix(filename):
     # Return success
     return 0
 
+
+def parse_shebang(shebangline):
+    end = len(shebangline)
```

Comment:
Please add a comment that on Linux, the first whitespace (U+0020) indicates the beginning of options.